packages/roosterjs-editor-core/lib/corePlugins/EntityPlugin.ts

```diff
@@ -236,6 +241,14 @@ export default class EntityPlugin implements PluginWithState<EntityPluginState>
                 if (element.shadowRoot) {
                     this.triggerEvent(element, EntityOperation.RemoveShadowRoot);
                 }
+
+                if (
+                    !shouldNormalizeDelimiters &&
+                    !element.isContentEditable &&
+                    getTagOfNode(element) === 'SPAN'
```

Comment:
Why check SPAN?

Reply:
To only handle the normalization if the deleted entity is readonly and inline

Reply:
Agree, that might be better.